INFO - 2023-09-28 18:32:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:32:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2023-09-28 18:32:29 --> Attempt to read property "login_id" on null
in APPPATH\Modules\Admin\Models\LoginModel.php on line 32.
 1 APPPATH\Modules\Admin\Models\LoginModel.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "login_id" on null', 'C:\\wamp64\\www\\modelo\\app\\Modules\\Admin\\Models\\LoginModel.php', 32)
 2 APPPATH\Modules\Admin\Controllers\Dashboard.php(43): App\Modules\Admin\Models\LoginModel->login('josue', '123')
 3 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->login()
 4 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
 5 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-28 18:32:29 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\modelo\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\modelo\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\m...')
#3 C:\wamp64\www\modelo\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Attempt to read property "login_id" on null
#0 C:\wamp64\www\modelo\app\Modules\Admin\Models\LoginModel.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read...', 'C:\\wamp64\\www\\m...', 32)
#1 C:\wamp64\www\modelo\app\Modules\Admin\Controllers\Dashboard.php(43): App\Modules\Admin\Models\LoginModel->login('josue', '123')
#2 C:\wamp64\www\modelo\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->login()
#3 C:\wamp64\www\modelo\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
#4 C:\wamp64\www\modelo\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\wamp64\www\modelo\index.php(79): CodeIgniter\CodeIgniter->run()
#6 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2023-09-28 18:32:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:33:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:33:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:33:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:34:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:36:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:36:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2023-09-28 18:36:24 --> Attempt to read property "login_id" on null
in APPPATH\Modules\Admin\Models\LoginModel.php on line 32.
 1 APPPATH\Modules\Admin\Models\LoginModel.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "login_id" on null', 'C:\\wamp64\\www\\modelo\\app\\Modules\\Admin\\Models\\LoginModel.php', 32)
 2 APPPATH\Modules\Admin\Controllers\Dashboard.php(43): App\Modules\Admin\Models\LoginModel->login('admin', 'dmin123')
 3 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->login()
 4 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
 5 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-28 18:36:24 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\modelo\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\modelo\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\m...')
#3 C:\wamp64\www\modelo\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Attempt to read property "login_id" on null
#0 C:\wamp64\www\modelo\app\Modules\Admin\Models\LoginModel.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read...', 'C:\\wamp64\\www\\m...', 32)
#1 C:\wamp64\www\modelo\app\Modules\Admin\Controllers\Dashboard.php(43): App\Modules\Admin\Models\LoginModel->login('admin', 'dmin123')
#2 C:\wamp64\www\modelo\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->login()
#3 C:\wamp64\www\modelo\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
#4 C:\wamp64\www\modelo\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\wamp64\www\modelo\index.php(79): CodeIgniter\CodeIgniter->run()
#6 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2023-09-28 18:36:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2023-09-28 18:36:33 --> Attempt to read property "login_id" on null
in APPPATH\Modules\Admin\Models\LoginModel.php on line 32.
 1 APPPATH\Modules\Admin\Models\LoginModel.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "login_id" on null', 'C:\\wamp64\\www\\modelo\\app\\Modules\\Admin\\Models\\LoginModel.php', 32)
 2 APPPATH\Modules\Admin\Controllers\Dashboard.php(43): App\Modules\Admin\Models\LoginModel->login('admin', 'dmin123')
 3 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->login()
 4 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
 5 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-28 18:36:33 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\modelo\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\modelo\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\m...')
#3 C:\wamp64\www\modelo\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Attempt to read property "login_id" on null
#0 C:\wamp64\www\modelo\app\Modules\Admin\Models\LoginModel.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read...', 'C:\\wamp64\\www\\m...', 32)
#1 C:\wamp64\www\modelo\app\Modules\Admin\Controllers\Dashboard.php(43): App\Modules\Admin\Models\LoginModel->login('admin', 'dmin123')
#2 C:\wamp64\www\modelo\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Dashboard->login()
#3 C:\wamp64\www\modelo\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Dashboard))
#4 C:\wamp64\www\modelo\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\wamp64\www\modelo\index.php(79): CodeIgniter\CodeIgniter->run()
#6 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2023-09-28 18:36:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:36:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:36:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:44:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:44:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:45:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:45:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:48:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:49:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:50:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:50:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2023-09-28 18:50:10 --> Undefined property: App\Modules\Admin\Controllers\Usuarios::$userModel
in APPPATH\Modules\Admin\Controllers\Usuarios.php on line 72.
 1 APPPATH\Modules\Admin\Controllers\Usuarios.php(72): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Modules\\Admin\\Controllers\\Usuarios::$userModel', 'C:\\wamp64\\www\\modelo\\app\\Modules\\Admin\\Controllers\\Usuarios.php', 72)
 2 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->table()
 3 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-28 18:50:10 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\modelo\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\modelo\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\m...')
#3 C:\wamp64\www\modelo\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Undefined property: App\Modules\Admin\Controllers\Usuarios::$userModel
#0 C:\wamp64\www\modelo\app\Modules\Admin\Controllers\Usuarios.php(72): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\wamp64\\www\\m...', 72)
#1 C:\wamp64\www\modelo\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Usuarios->table()
#2 C:\wamp64\www\modelo\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Usuarios))
#3 C:\wamp64\www\modelo\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\wamp64\www\modelo\index.php(79): CodeIgniter\CodeIgniter->run()
#5 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2023-09-28 18:52:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:52:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:52:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:52:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:53:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:53:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2023-09-28 18:53:11 --> Undefined property: App\Modules\Admin\Controllers\Users::$userModel
in APPPATH\Modules\Admin\Controllers\Users.php on line 72.
 1 APPPATH\Modules\Admin\Controllers\Users.php(72): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Modules\\Admin\\Controllers\\Users::$userModel', 'C:\\wamp64\\www\\modelo\\app\\Modules\\Admin\\Controllers\\Users.php', 72)
 2 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->listUsers()
 3 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-28 18:53:11 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\modelo\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\modelo\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\m...')
#3 C:\wamp64\www\modelo\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Undefined property: App\Modules\Admin\Controllers\Users::$userModel
#0 C:\wamp64\www\modelo\app\Modules\Admin\Controllers\Users.php(72): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\wamp64\\www\\m...', 72)
#1 C:\wamp64\www\modelo\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->listUsers()
#2 C:\wamp64\www\modelo\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
#3 C:\wamp64\www\modelo\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\wamp64\www\modelo\index.php(79): CodeIgniter\CodeIgniter->run()
#5 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2023-09-28 18:55:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 18:55:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2023-09-28 18:55:16 --> syntax error, unexpected token "/"
in APPPATH\Modules\Admin\Controllers\Users.php on line 38.
 1 SYSTEMPATH\Autoloader\Autoloader.php(285): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\wamp64\\www\\modelo\\app\\Modules\\Admin\\Controllers\\Users.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(263): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Modules\\Admin\\Controllers\\Users')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Modules\\Admin\\Controllers\\Users')
 4 SYSTEMPATH\CodeIgniter.php(873): class_exists('\\App\\Modules\\Admin\\Controllers\\Users', true)
 5 SYSTEMPATH\CodeIgniter.php(479): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-28 18:55:16 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\modelo\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\modelo\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ParseError), 500, 'C:\\wamp64\\www\\m...')
#3 C:\wamp64\www\modelo\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ParseError), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ParseError))
#5 {main}
  thrown
【Previous Exception】
ParseError
syntax error, unexpected token "/"
#0 C:\wamp64\www\modelo\system\Autoloader\Autoloader.php(285): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\wamp64\\www\\m...')
#1 C:\wamp64\www\modelo\system\Autoloader\Autoloader.php(263): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Modules\\Adm...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Modules\\Adm...')
#3 C:\wamp64\www\modelo\system\CodeIgniter.php(873): class_exists('\\App\\Modules\\Ad...', true)
#4 C:\wamp64\www\modelo\system\CodeIgniter.php(479): CodeIgniter\CodeIgniter->startController()
#5 C:\wamp64\www\modelo\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\wamp64\www\modelo\index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2023-09-28 18:55:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2023-09-28 18:55:43 --> Undefined property: App\Modules\Admin\Controllers\Users::$userModel
in APPPATH\Modules\Admin\Controllers\Users.php on line 73.
 1 APPPATH\Modules\Admin\Controllers\Users.php(73): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Modules\\Admin\\Controllers\\Users::$userModel', 'C:\\wamp64\\www\\modelo\\app\\Modules\\Admin\\Controllers\\Users.php', 73)
 2 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->listUsers()
 3 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-28 18:55:43 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\modelo\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\modelo\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\m...')
#3 C:\wamp64\www\modelo\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Undefined property: App\Modules\Admin\Controllers\Users::$userModel
#0 C:\wamp64\www\modelo\app\Modules\Admin\Controllers\Users.php(73): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\wamp64\\www\\m...', 73)
#1 C:\wamp64\www\modelo\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->listUsers()
#2 C:\wamp64\www\modelo\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
#3 C:\wamp64\www\modelo\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\wamp64\www\modelo\index.php(79): CodeIgniter\CodeIgniter->run()
#5 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2023-09-28 18:58:28 --> syntax error, unexpected token "default"
in APPPATH\Modules\Admin\Controllers\Users.php on line 71.
 1 SYSTEMPATH\Autoloader\Autoloader.php(285): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\wamp64\\www\\modelo\\app\\Modules\\Admin\\Controllers\\Users.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(263): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Modules\\Admin\\Controllers\\Users')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Modules\\Admin\\Controllers\\Users')
 4 SYSTEMPATH\CodeIgniter.php(873): class_exists('\\App\\Modules\\Admin\\Controllers\\Users', true)
 5 SYSTEMPATH\CodeIgniter.php(479): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-28 18:58:28 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\modelo\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\modelo\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ParseError), 500, 'C:\\wamp64\\www\\m...')
#3 C:\wamp64\www\modelo\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ParseError), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ParseError))
#5 {main}
  thrown
【Previous Exception】
ParseError
syntax error, unexpected token "default"
#0 C:\wamp64\www\modelo\system\Autoloader\Autoloader.php(285): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\wamp64\\www\\m...')
#1 C:\wamp64\www\modelo\system\Autoloader\Autoloader.php(263): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Modules\\Adm...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Modules\\Adm...')
#3 C:\wamp64\www\modelo\system\CodeIgniter.php(873): class_exists('\\App\\Modules\\Ad...', true)
#4 C:\wamp64\www\modelo\system\CodeIgniter.php(479): CodeIgniter\CodeIgniter->startController()
#5 C:\wamp64\www\modelo\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\wamp64\www\modelo\index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2023-09-28 18:58:29 --> syntax error, unexpected token "default"
in APPPATH\Modules\Admin\Controllers\Users.php on line 71.
 1 SYSTEMPATH\Autoloader\Autoloader.php(285): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\wamp64\\www\\modelo\\app\\Modules\\Admin\\Controllers\\Users.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(263): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Modules\\Admin\\Controllers\\Users')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Modules\\Admin\\Controllers\\Users')
 4 SYSTEMPATH\CodeIgniter.php(873): class_exists('\\App\\Modules\\Admin\\Controllers\\Users', true)
 5 SYSTEMPATH\CodeIgniter.php(479): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-28 18:58:29 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\modelo\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\modelo\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ParseError), 500, 'C:\\wamp64\\www\\m...')
#3 C:\wamp64\www\modelo\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ParseError), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ParseError))
#5 {main}
  thrown
【Previous Exception】
ParseError
syntax error, unexpected token "default"
#0 C:\wamp64\www\modelo\system\Autoloader\Autoloader.php(285): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\wamp64\\www\\m...')
#1 C:\wamp64\www\modelo\system\Autoloader\Autoloader.php(263): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Modules\\Adm...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Modules\\Adm...')
#3 C:\wamp64\www\modelo\system\CodeIgniter.php(873): class_exists('\\App\\Modules\\Ad...', true)
#4 C:\wamp64\www\modelo\system\CodeIgniter.php(479): CodeIgniter\CodeIgniter->startController()
#5 C:\wamp64\www\modelo\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\wamp64\www\modelo\index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2023-09-28 18:59:11 --> syntax error, unexpected token "default"
in APPPATH\Modules\Admin\Controllers\Users.php on line 71.
 1 SYSTEMPATH\Autoloader\Autoloader.php(285): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\wamp64\\www\\modelo\\app\\Modules\\Admin\\Controllers\\Users.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(263): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Modules\\Admin\\Controllers\\Users')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Modules\\Admin\\Controllers\\Users')
 4 SYSTEMPATH\CodeIgniter.php(873): class_exists('\\App\\Modules\\Admin\\Controllers\\Users', true)
 5 SYSTEMPATH\CodeIgniter.php(479): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-28 18:59:11 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\modelo\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\modelo\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ParseError), 500, 'C:\\wamp64\\www\\m...')
#3 C:\wamp64\www\modelo\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ParseError), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ParseError))
#5 {main}
  thrown
【Previous Exception】
ParseError
syntax error, unexpected token "default"
#0 C:\wamp64\www\modelo\system\Autoloader\Autoloader.php(285): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\wamp64\\www\\m...')
#1 C:\wamp64\www\modelo\system\Autoloader\Autoloader.php(263): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Modules\\Adm...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Modules\\Adm...')
#3 C:\wamp64\www\modelo\system\CodeIgniter.php(873): class_exists('\\App\\Modules\\Ad...', true)
#4 C:\wamp64\www\modelo\system\CodeIgniter.php(479): CodeIgniter\CodeIgniter->startController()
#5 C:\wamp64\www\modelo\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\wamp64\www\modelo\index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2023-09-28 18:59:29 --> Unmatched '}'
in APPPATH\Modules\Admin\Controllers\Users.php on line 79.
 1 SYSTEMPATH\Autoloader\Autoloader.php(285): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\wamp64\\www\\modelo\\app\\Modules\\Admin\\Controllers\\Users.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(263): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Modules\\Admin\\Controllers\\Users')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Modules\\Admin\\Controllers\\Users')
 4 SYSTEMPATH\CodeIgniter.php(873): class_exists('\\App\\Modules\\Admin\\Controllers\\Users', true)
 5 SYSTEMPATH\CodeIgniter.php(479): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-28 18:59:29 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\modelo\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\modelo\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ParseError), 500, 'C:\\wamp64\\www\\m...')
#3 C:\wamp64\www\modelo\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ParseError), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ParseError))
#5 {main}
  thrown
【Previous Exception】
ParseError
Unmatched '}'
#0 C:\wamp64\www\modelo\system\Autoloader\Autoloader.php(285): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\wamp64\\www\\m...')
#1 C:\wamp64\www\modelo\system\Autoloader\Autoloader.php(263): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Modules\\Adm...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Modules\\Adm...')
#3 C:\wamp64\www\modelo\system\CodeIgniter.php(873): class_exists('\\App\\Modules\\Ad...', true)
#4 C:\wamp64\www\modelo\system\CodeIgniter.php(479): CodeIgniter\CodeIgniter->startController()
#5 C:\wamp64\www\modelo\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\wamp64\www\modelo\index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2023-09-28 19:00:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:12:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:12:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:13:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2023-09-28 19:13:51 --> Undefined property: App\Modules\Admin\Controllers\Users::$userModel
in APPPATH\Modules\Admin\Controllers\Users.php on line 73.
 1 APPPATH\Modules\Admin\Controllers\Users.php(73): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Modules\\Admin\\Controllers\\Users::$userModel', 'C:\\wamp64\\www\\modelo\\app\\Modules\\Admin\\Controllers\\Users.php', 73)
 2 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->listUsers()
 3 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-28 19:13:51 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\modelo\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\modelo\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\m...')
#3 C:\wamp64\www\modelo\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Undefined property: App\Modules\Admin\Controllers\Users::$userModel
#0 C:\wamp64\www\modelo\app\Modules\Admin\Controllers\Users.php(73): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\wamp64\\www\\m...', 73)
#1 C:\wamp64\www\modelo\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->listUsers()
#2 C:\wamp64\www\modelo\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
#3 C:\wamp64\www\modelo\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\wamp64\www\modelo\index.php(79): CodeIgniter\CodeIgniter->run()
#5 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2023-09-28 19:17:04 --> Class "App\Modules\Admin\Models\UserModel" not found
in APPPATH\Modules\Admin\Controllers\Users.php on line 22.
 1 SYSTEMPATH\CodeIgniter.php(887): App\Modules\Admin\Controllers\Users->__construct()
 2 SYSTEMPATH\CodeIgniter.php(483): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-28 19:17:04 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\modelo\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\modelo\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\wamp64\\www\\m...')
#3 C:\wamp64\www\modelo\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#5 {main}
  thrown
【Previous Exception】
Error
Class "App\Modules\Admin\Models\UserModel" not found
#0 C:\wamp64\www\modelo\system\CodeIgniter.php(887): App\Modules\Admin\Controllers\Users->__construct()
#1 C:\wamp64\www\modelo\system\CodeIgniter.php(483): CodeIgniter\CodeIgniter->createController()
#2 C:\wamp64\www\modelo\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\wamp64\www\modelo\index.php(79): CodeIgniter\CodeIgniter->run()
#4 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2023-09-28 19:20:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:23:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:23:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:26:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:27:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:34:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2023-09-28 19:34:59 --> Undefined property: stdClass::$id
in APPPATH\Views\admin\tables.php on line 43.
 1 APPPATH\Views\admin\tables.php(43): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\wamp64\\www\\modelo\\app\\Views\\admin\\tables.php', 43)
 2 SYSTEMPATH\View\View.php(226): include('C:\\wamp64\\www\\modelo\\app\\Views\\admin\\tables.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('admin/tables', [], true)
 5 APPPATH\Modules\Admin\Controllers\Users.php(78): view('admin/tables', [...])
 6 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->listUsers()
 7 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-28 19:34:59 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\modelo\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\modelo\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\m...')
#3 C:\wamp64\www\modelo\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Undefined property: stdClass::$id
#0 C:\wamp64\www\modelo\app\Views\admin\tables.php(43): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\wamp64\\www\\m...', 43)
#1 C:\wamp64\www\modelo\system\View\View.php(226): include('C:\\wamp64\\www\\m...')
#2 C:\wamp64\www\modelo\system\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\wamp64\www\modelo\system\Common.php(1178): CodeIgniter\View\View->render('admin/tables', Array, true)
#4 C:\wamp64\www\modelo\app\Modules\Admin\Controllers\Users.php(78): view('admin/tables', Array)
#5 C:\wamp64\www\modelo\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->listUsers()
#6 C:\wamp64\www\modelo\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
#7 C:\wamp64\www\modelo\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp64\www\modelo\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2023-09-28 19:36:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:37:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:37:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:40:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2023-09-28 19:40:05 --> Undefined property: stdClass::$id
in APPPATH\Views\admin\listUsers.php on line 43.
 1 APPPATH\Views\admin\listUsers.php(43): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\wamp64\\www\\modelo\\app\\Views\\admin\\listUsers.php', 43)
 2 SYSTEMPATH\View\View.php(226): include('C:\\wamp64\\www\\modelo\\app\\Views\\admin\\listUsers.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('admin/listUsers', [], true)
 5 APPPATH\Modules\Admin\Controllers\Users.php(78): view('admin/listUsers', [...])
 6 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->listUsers()
 7 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-28 19:40:05 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\modelo\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\modelo\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\m...')
#3 C:\wamp64\www\modelo\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Undefined property: stdClass::$id
#0 C:\wamp64\www\modelo\app\Views\admin\listUsers.php(43): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\wamp64\\www\\m...', 43)
#1 C:\wamp64\www\modelo\system\View\View.php(226): include('C:\\wamp64\\www\\m...')
#2 C:\wamp64\www\modelo\system\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\wamp64\www\modelo\system\Common.php(1178): CodeIgniter\View\View->render('admin/listUsers', Array, true)
#4 C:\wamp64\www\modelo\app\Modules\Admin\Controllers\Users.php(78): view('admin/listUsers', Array)
#5 C:\wamp64\www\modelo\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->listUsers()
#6 C:\wamp64\www\modelo\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
#7 C:\wamp64\www\modelo\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp64\www\modelo\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2023-09-28 19:40:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:40:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2023-09-28 19:40:09 --> Undefined property: stdClass::$id
in APPPATH\Views\admin\listUsers.php on line 43.
 1 APPPATH\Views\admin\listUsers.php(43): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\wamp64\\www\\modelo\\app\\Views\\admin\\listUsers.php', 43)
 2 SYSTEMPATH\View\View.php(226): include('C:\\wamp64\\www\\modelo\\app\\Views\\admin\\listUsers.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('admin/listUsers', [], true)
 5 APPPATH\Modules\Admin\Controllers\Users.php(78): view('admin/listUsers', [...])
 6 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->listUsers()
 7 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-28 19:40:09 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\modelo\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\modelo\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\m...')
#3 C:\wamp64\www\modelo\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Undefined property: stdClass::$id
#0 C:\wamp64\www\modelo\app\Views\admin\listUsers.php(43): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\wamp64\\www\\m...', 43)
#1 C:\wamp64\www\modelo\system\View\View.php(226): include('C:\\wamp64\\www\\m...')
#2 C:\wamp64\www\modelo\system\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\wamp64\www\modelo\system\Common.php(1178): CodeIgniter\View\View->render('admin/listUsers', Array, true)
#4 C:\wamp64\www\modelo\app\Modules\Admin\Controllers\Users.php(78): view('admin/listUsers', Array)
#5 C:\wamp64\www\modelo\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->listUsers()
#6 C:\wamp64\www\modelo\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
#7 C:\wamp64\www\modelo\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp64\www\modelo\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2023-09-28 19:40:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:42:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2023-09-28 19:42:52 --> Undefined property: stdClass::$funcioanrio_id
in APPPATH\Views\admin\listUsers.php on line 35.
 1 APPPATH\Views\admin\listUsers.php(35): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$funcioanrio_id', 'C:\\wamp64\\www\\modelo\\app\\Views\\admin\\listUsers.php', 35)
 2 SYSTEMPATH\View\View.php(226): include('C:\\wamp64\\www\\modelo\\app\\Views\\admin\\listUsers.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('admin/listUsers', [], true)
 5 APPPATH\Modules\Admin\Controllers\Users.php(78): view('admin/listUsers', [...])
 6 SYSTEMPATH\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->listUsers()
 7 SYSTEMPATH\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-28 19:42:52 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\modelo\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\modelo\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\wamp64\\www\\m...')
#3 C:\wamp64\www\modelo\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
Undefined property: stdClass::$funcioanrio_id
#0 C:\wamp64\www\modelo\app\Views\admin\listUsers.php(35): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\wamp64\\www\\m...', 35)
#1 C:\wamp64\www\modelo\system\View\View.php(226): include('C:\\wamp64\\www\\m...')
#2 C:\wamp64\www\modelo\system\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\wamp64\www\modelo\system\Common.php(1178): CodeIgniter\View\View->render('admin/listUsers', Array, true)
#4 C:\wamp64\www\modelo\app\Modules\Admin\Controllers\Users.php(78): view('admin/listUsers', Array)
#5 C:\wamp64\www\modelo\system\CodeIgniter.php(915): App\Modules\Admin\Controllers\Users->listUsers()
#6 C:\wamp64\www\modelo\system\CodeIgniter.php(492): CodeIgniter\CodeIgniter->runController(Object(App\Modules\Admin\Controllers\Users))
#7 C:\wamp64\www\modelo\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp64\www\modelo\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2023-09-28 19:43:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:45:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:45:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:47:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:54:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:54:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:57:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:57:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:57:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:57:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:57:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 19:57:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
CRITICAL - 2023-09-28 20:07:06 --> syntax error, unexpected token "}", expecting "," or ";"
in APPPATH\Modules\Land\Controllers\Home.php on line 10.
 1 SYSTEMPATH\Autoloader\Autoloader.php(285): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\wamp64\\www\\modelo\\app\\Modules\\Land\\Controllers\\Home.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(263): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Modules\\Land\\Controllers\\Home')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Modules\\Land\\Controllers\\Home')
 4 SYSTEMPATH\CodeIgniter.php(873): class_exists('\\App\\Modules\\Land\\Controllers\\Home', true)
 5 SYSTEMPATH\CodeIgniter.php(479): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-09-28 20:07:06 --> Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\wamp64\www\modelo\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(241): include()
#1 C:\wamp64\www\modelo\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\wamp64\www\modelo\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ParseError), 500, 'C:\\wamp64\\www\\m...')
#3 C:\wamp64\www\modelo\system\Debug\Exceptions.php(147): CodeIgniter\Debug\ExceptionHandler->handle(Object(ParseError), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ParseError))
#5 {main}
  thrown
【Previous Exception】
ParseError
syntax error, unexpected token "}", expecting "," or ";"
#0 C:\wamp64\www\modelo\system\Autoloader\Autoloader.php(285): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\wamp64\\www\\m...')
#1 C:\wamp64\www\modelo\system\Autoloader\Autoloader.php(263): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Modules\\Lan...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Modules\\Lan...')
#3 C:\wamp64\www\modelo\system\CodeIgniter.php(873): class_exists('\\App\\Modules\\La...', true)
#4 C:\wamp64\www\modelo\system\CodeIgniter.php(479): CodeIgniter\CodeIgniter->startController()
#5 C:\wamp64\www\modelo\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\wamp64\www\modelo\index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2023-09-28 20:14:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 20:14:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 20:15:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 20:16:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 20:16:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 20:16:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 20:17:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 20:17:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 20:18:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
INFO - 2023-09-28 20:18:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\DatabaseHandler' driver.
